### PR TITLE
Fix various outdated links and link formatting in docs.

### PR DESCRIPTION
**Pull request by HumairAK**
This docs update various links in our docs, types of changes can be summed up as follows: 

- Update outdated links to point to their proper locations
- Change remote links to their relative locations where it applies
- Convert inline links to ref format, currently our docs used a mixture, this makes it consistent throughout. 

Resolves: https://github.com/operate-first/apps/issues/1412

### CODE REVIEW

File: docs/content/README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Operate First manages various applications and services in the environments list
 [10]: https://github.com/operate-first/support/issues/new?assignees=first-operator&labels=kind%2Fonboarding%2Carea%2Fbucket&template=ceph_bucket_request.yaml&title=BUCKET%3A+%3Cname%3E
 [11]: https://github.com/operate-first/apps/tree/master/dex
 [12]: https://rook.io/
-[13]: https://www.operate-first.cloud/users/support/docs/claiming_object_store.md
+[13]: ocs/claiming_object_store.md
```
Comment (dystewart) [inline docs/content/README.md:82]: Part of link missing on lines 82 and 95?

File: docs/content/argocd-gitops/encrypting_applications.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
-In the link above you will require a public fingerprint to include in your sops.yaml, this comes from a gpg key we use for all our apps in this repo and can be found [here](https://keys.openpgp.org/search?q=aicoe-operate-first%40redhat.com).
+In the link above you will require a public fingerprint to include in your sops.yaml, this comes from a gpg key we use
+for all our apps in this repo and can be found [here][op1stkey].
 
-If you are an active contributor and require the private key, please make an issue or reach out to another active contributor/maintainer for it. Be prepared to provide your own public key so that we may send it to you securely.
+If you are an active contributor and require the private key, please make an issue or reach out to another active
+contributor/maintainer for it. Be prepared to provide your own public key so that we may send it to you securely.
+
+
+[ksopsreadme]: https://github.com/viaduct-ai/kustomize-sops#installation
```
Comment (Gregory-Pereira) [inline docs/content/argocd-gitops/encrypting_applications.md:16]: we should keep this consistent between using numeric and word-based link names

File: docs/content/grafana/README.md [hunk 3]
```diff
@@ -1,28 +1,38 @@
...
 Dashboards are exported in Grafana JSON format, and contain everything you need (layout, variables, styles, data sources, queries, etc) to import the dashboard at a later time.
 
-You can import these dashboards in Grafana either by pasting the dashboard JSON text directly into the text area or directly uploading the JSON file. Make sure to add/connect your [data sources](https://grafana.com/docs/grafana/latest/datasources/) to Grafana and pick what data source you want the dashboard to use.
+You can import these dashboards in Grafana either by pasting the dashboard JSON text directly into the text area or directly uploading the JSON file. Make sure to add/connect your [data sources][8] to Grafana and pick what data source you want the dashboard to use.
 
-For more details on importing/exporting dashboards, you can refer to the Grafana documentation [here](https://grafana.com/docs/grafana/latest/dashboards/export-import/).
+For more details on importing/exporting dashboards, you can refer to the Grafana documentation [here][9].
+
+[1]: https://grafana.com/oss/grafana
+[2]: https://github.com/operate-first/apps/tree/master/grafana/overlays/moc/smaug/dashboards
```
Comment (Gregory-Pereira) [inline docs/content/grafana/README.md:31]: We should be consistent between relative and absolute links.

File: docs/content/odh/jupyterhub/analyze_storage.md [hunk 4]
```diff
@@ -22,7 +21,8 @@ du -h --max-depth=2 . | sort -h
 This should display a list of folders on your PVC sorted by their size.
 
 
-
 ## How can I cleanup my `Pipenv` cache?
 
 Simply delete the folder named `.cache/pipenv`.
+
+[1]: https://grafana.operate-first.cloud/d/fuJBFErMz/jupyterhub-user-perspective?orgId=1
```
Comment (Gregory-Pereira) [inline docs/content/odh/jupyterhub/analyze_storage.md:28]: Link here but no link usage in diff

**Review by Gregory-Pereira — APPROVED** [review]
/Igtm, ignore the two comments about relative vs static and numeric vs word-based link paths since it has been discussed to convert them after the fact.

File: docs/content/odh/jupyterhub/analyze_storage.md (hunk 4 above)
Comment (HumairAK) [reply docs/content/odh/jupyterhub/analyze_storage.md:28]: good catch! fixed.

**Review by harshad16 — APPROVED** [review]
/lgtm

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**